spec/fabricators/topic_fabricator.rb

```diff
@@ -20,6 +20,14 @@
   archetype Archetype.banner
 end
 
+Fabricator(:with_posts_topic, from: :topic) do
```

Comment:
You are the native english speaker, so I might be wrong, but it reads weirdly to me.

`Fabricate(:topic_with_posts)` ?

Reply:
Yeah that is better not sure why I did it backwards